### ISSUE TITLE
Grouped fields

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#8104]
iD has a few special fields that group related tags together, like Name, Address, Structure, Allowed Access, and Bike Lanes. Grouping is a nice way to make the inspector simpler and more intuitive. There are a lot more tags that could benefit from grouping, for example:

- Internet connection/access/network name
- Wheelchair access/description (#7698)
- Population number/source/date (#6672)
- Fee presence/amount/payment types

Instead of coding a special field type for every grouping, we should look into a reusable grouped field type that can handle simple combinations.

**Comment by karmanya007** [comment]
@quincylvania Hey, I want to start contributing to this project. Could you guide me a bit more on this issue. Would love to start off somewhere.

**Comment by quincylvania** [comment]
Welcome, @karmanya007. This issue is actually a bit advanced, and there aren't that many introductory issues open right now. But if you're feeling intrepid then can read our [contributing guide](https://github.com/openstreetmap/iD/blob/develop/CONTRIBUTING.md) and jump in anywhere.

**Comment by maro-21** [comment]
> Instead of coding a special field type for every grouping, we should look into a reusable grouped field type that can handle simple combinations.

We already have such fields - like operator and operator:type for schools. Can't we use the same for the ones above? If a user fills in the first field, other appear below it.

**Comment by quincylvania** [comment]
> We already have such fields - like operator and operator:type for schools.

@maro-21 Yes, these already work such that one field's appearance depends on a related field's value, but they should also be grouped together visually.

**Comment by karmanya007** [comment]
@quincylvania I think, now I can take this one. Could you tell me how the grouping happens?

**Comment by danieldegroot2** [comment]
also
* building levels / underground building levels
(`building:levels` / `building:levels:underground`)
